docs/quickstart/tornjak-configmap.yaml

```
@@ -5,16 +5,25 @@ metadata:
   namespace: spire
 data:
   server.conf: |
+
     server {
-      metadata = "insert metadata"
+      # location of SPIRE socket
+      # here, set to default SPIRE socket path
+      spire_socket_path = "unix:///tmp/spire-server/private/api.sock"
+
+      # configure HTTP connection to Tornjak server
+      http {
+        enabled = true
+        #port = 10000 # opens at port 10000
```

```suggestion
        port = 10000 # opens at port 10000
```
Shouldn't we have this service active by default?